omero/sysadmins/unix/install-web.txt

```diff
@@ -29,9 +29,9 @@ There are a number of good ways to easily deploy OMERO.web.
 
 Deploying OMERO.web separately from OMERO.server is recommended as they
 perform best under different circumstances and require a different set of
-dependencies. Please consider using :doc:`OMERO.web trial deployment <install-web/install-web-trial>`.
+dependencies. Please consider using :doc:`OMERO.web WSGI deployment <install-web/web-deployment>`.
```

Comment:
Again, let's remove the hard coded link text and let the build automatically use the page title